docs/conf.py

```diff
@@ -56,11 +53,15 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
-# The short X.Y version.
-version = '4.1.0'
+
 # The full version, including alpha/beta/rc tags.
-release = '4.1.0'
+import io
+with io.open('../version.txt') as f:
+    release = f.read().strip()
```

Comment:
It's possible to build the docs without having the package installed, so pkg_resources can't be relied upon. Granted the `.. autoXXX` stuff may (?) not work as desired if the package isn't installed, but that is an option, and presumably it's a common enough one that RTD doesn't install the package by default. So I prefer not to have conf.py depend on the package being installed, and an external version.txt is one way to do that while also DRY.